progress/README.md

```diff
@@ -0,0 +1,305 @@
+# Progress
+
+## Requests
+
+|                     Route                     |                                          Description                                          |
+| :-------------------------------------------: | :-------------------------------------------------------------------------------------------: |
+|     [POST /progresses](#post-progresses)      |                                 Creates a new progress entry.                                 |
+|      [GET /progresses](#get-progresses)       |                 Retrieves progress entries based on the provided parameters.                  |
+| [GET /progresses/range](#get-progressesrange) | Retrieves the progress records for a particular user or task within the specified date range. |
+
+## POST /progresses
+
+    Creates a new progress entry.
+
+- **Params**  
+  None
+- **Query**
+  None
+- **Body**  
+  Attributes:
+  -type (required, string): Specifies the type of the progress entry (e.g., "task", "user").
+  -taskId (optional, string): Task ID associated with the progress entry (applicable for task progress only).
+  -completed (required, string): Completed portion of the task or progress.
+  -planned (required, string): Planned portion of the task or progress.
+  -blockers (required, string): Blockers for the progress entry.
+- **Headers**  
+  Content-Type: application/json
```

Comment:
Please mention that since it is an authenticated route we wont need user id as a parameter.

Reply:
Mentioned. Thanks for highlighting.